src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -494,3 +494,8 @@ public net.minecraft.world.item.enchantment.Enchantment locationContext(Lnet/min
 public net.minecraft.world.item.enchantment.Enchantment entityContext(Lnet/minecraft/server/level/ServerLevel;ILnet/minecraft/world/entity/Entity;Lnet/minecraft/world/phys/Vec3;)Lnet/minecraft/world/level/storage/loot/LootContext;
 public net.minecraft.world.item.enchantment.Enchantment blockHitContext(Lnet/minecraft/server/level/ServerLevel;ILnet/minecraft/world/entity/Entity;Lnet/minecraft/world/phys/Vec3;Lnet/minecraft/world/level/block/state/BlockState;)Lnet/minecraft/world/level/storage/loot/LootContext;
 public net.minecraft.world.item.enchantment.Enchantment applyEffects(Ljava/util/List;Lnet/minecraft/world/level/storage/loot/LootContext;Ljava/util/function/Consumer;)V
+
+# Registries and Registry Codecs
```

Comment:
This shouldn't be necessary, there are already APIs to unwrap the registry ops ;)

Reply:
Which APIs?